lua/rainbow-delimiters/lib.lua

```diff
@@ -64,7 +64,7 @@ M.buffers = {}
 ---Fetches the query object for the given language from the settings.
 ---
 ---@param lang string  Name of the language to get the query for
----@return userdata query  The query object
+---@return Query|nil query  The query object
```

Comment:
```suggestion
---@return Query? query  The query object
```
I can edit this later, but didn't want to do it while you might be editing. Just for consistency, this should be with `?` instead of `|nil`.